canonical-websites/www.ubuntu.com
# Added redirect from /cloud/openstack/autopilot to /openstack

## Done

- Added redirect from /cloud/openstack/autopilot to /openstack
- Added issue to fix landscape.canonical.com where people are coming from

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [cloud/openstack/autopilot/](http://0.0.0.0:8001/cloud/openstack/autopilot/) and see that it takes you to /openstack

## Issue / Card

Fixes #2927
